### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by nicolasmaia** [bbqsrc/sax#1]
A LICENSE.md, and preferably license headers on files :)

**Comment by bbqsrc** [comment]
Yes. I haven't decided which one I'm going to use yet though, hence **All Rights Reserved** being strewn across all code and the sound of sirens in the distance.

Opinionated suggestions welcomed.

**Comment by ninks** [comment]
GPL2 or GPL3 are good free (freedom) respectling lisences. Made by Free Software Foundation.

**Comment by bbqsrc** [comment]
Yes, I would be choosing between GPLv3 or ISC/BSD-2-Clause ultimately.

Ultimately comes down to which the community lets me know they prefer, I think.

**Comment by nicolasmaia** [comment]
I'm all for GPLv3. Perhaps even AGPLv3+, if we're feeling generous :)

**Comment by ghost** [comment]
I have no idea how the AGPLv3 would be relevant to this software.

AGPLv3 is for:

* SAAS
* libraries
* software that might get linked with other software

This software is none of the above. This entire thread is a GPL troll.

**Comment by bbqsrc** [comment]
AGPL differs to GPL only in its proliferation clause ensuring that network deployment such as SaaS, or a website, etc requires linking to the source of that instance. In all other ways it is the same as the GPL. In this case, as a desktop app, there is no point using the AGPL at all.

Insofar as I am concerned, using the GPL does create an incentive to stop cheap bastards from coming along, making one small modification to a codebase and then going and profiteering from the hard work of another community without any reciprocation. Pieter Hintjens has written about this in [How To Capture an Open Source Community](http://hintjens.com/blog:68) and [How to Make Money from Open Source](http://hintjens.com/blog:27).

I suspect ultimately, for this project, I am tossing up between using GPLv3 or the EUPL 1.1, which is [compared here](https://joinup.ec.europa.eu/community/eupl/news/eupl-or-gplv3-comparison-table-main-characteristics-and-differences).

Broadly, the EUPL is shorter and translated into 22 languages, and its enforceability is not in question especially in Europe. In particular, the jurisdiction is expressly set to Belgium if a person does not operate within the EU.

**Comment by nicolasmaia** [comment]
>In this case, as a desktop app, there is no point using the AGPL at all.

However isn't it the case that more and more often desktop apps are being ported to the web?

**Comment by bbqsrc** [comment]
This is a **client** application, not a **server** application. The provision has no useful impact in this case.
